### PR TITLE
Align metrics to interop

### PR DESCRIPTION
**Pull request by terencechain**
This PR aligns metrics to: https://github.com/ethereum/eth2.0-metrics/pull/9

### CODE REVIEW

File: shared/bytesutil/bytes.go [hunk 1]
```diff
@@ -142,3 +142,9 @@ func Trunc(x []byte) []byte {
 	}
 	return x
 }
+
+// ToLowU64 returns the lowest 8 bytes interpreted as little endian.
+func ToLowU64(x []byte) uint64 {
+	x = x[:8]
```
Comment (prestonvanloon) [inline shared/bytesutil/bytes.go]: Please handle byte slices less than len 8 (this panics).

File: shared/bytesutil/bytes.go [hunk 2]
```diff
@@ -142,3 +142,9 @@ func Trunc(x []byte) []byte {
 	}
 	return x
 }
+
+// ToLowU64 returns the lowest 8 bytes interpreted as little endian.
+func ToLowU64(x []byte) uint64 {
```
Comment (prestonvanloon) [inline shared/bytesutil/bytes.go]: This should be signed int 64, at least that is what sigma prime does.

File: shared/bytesutil/bytes.go (hunk 1 above)
Comment (terencechain) [reply shared/bytesutil/bytes.go]: done

File: shared/bytesutil/bytes.go (hunk 2 above)
Comment (terencechain) [reply shared/bytesutil/bytes.go]: done

File: shared/bytesutil/bytes.go [hunk 3]
```diff
@@ -142,3 +142,11 @@ func Trunc(x []byte) []byte {
 	}
 	return x
 }
+
+// ToLowU64 returns the lowest 8 bytes interpreted as little endian.
+func ToLowU64(x []byte) int64 {
```
Comment (prestonvanloon) [inline shared/bytesutil/bytes.go]: Please rename to `ToLowInt64` as it's not unsigned anymore

Comment (terencechain) [reply shared/bytesutil/bytes.go]: done @prestonvanloon

**Pull request merged**